docs/UserGuide.adoc

```diff
@@ -112,20 +112,45 @@ Searches for loans made by John Doe where the item loaned out is Bicycle-007
 
 === Deleting the chosen loan from the loan book : `delete`
 
-Delete the loan from the loan book. If you want to delete one loan, use `delete i/LOAN_ID`. If you want to delete all, use`delete!` +
-Format: `delete i/LOAN_ID` or `delete!`
+Delete the loan from the loan book. If you want to delete one loan, use `delete i/LOAN_ID x/CURRENT_PASSWORD`.
+As delete is a critical command, elevation is required hence the password field. +
```

Comment:
Perhaps you can elaborate on why it is critical: because it modifies transaction history.